scripts/rmc.py

```diff
@@ -31,11 +59,29 @@ def delete_file(filename):
     except OSError:
         pass
 
+# Add a set of CBMC flags to the CBMC arguments
+def add_set_cbmc_flags(args, flags):
+    # We print a warning if the user has passed the flag via `cbmc_args`
+    # Otherwise we append it to the CBMC arguments
+    for arg in flags:
+        # This behavior must be reviewed if the set of flags is extended
+        if arg in args.cbmc_args:
+            print("WARNING: Default CBMC argument `{}` not added (already specified)".format(arg))
```

Comment:
might be useful to also print the `cbmc_args` here

Reply:
Have to disagree with this. I expect the user to make a dry run if he really wants to see them. Plus, at this point `cbmc_args` is not complete (there may be other flags to add) so it could be misleading.